lib/shopify-cli/tasks/ensure_env.rb

```diff
@@ -21,6 +21,7 @@ def ask
           api_key: api_key,
           secret: api_secret,
           shop: shop,
+          host: @ctx.app_metadata[:host],
```

Comment:
I want to stop using this `app_metadata` mostly because it's confusing. It doesn't persist so I'm not sure it will be populated here in all cases. The safe thing to do is to call `Tasks::Tunnel.start` which will return the url or start a tunnel and return the url.

Reply:
Oh! cool okay.